### ISSUE TITLE
cannot interact with very bottom of map (or buttons) on iPhone X

### ISSUE THREAD

**Issue by caleb87** [mapsplugin/cordova-plugin-googlemaps#2098]
**I'm submitting a ...**  (check one with "x")
- [ ] question
- [x] any problem or bug report
- [ ] feature request

**If you choose 'problem or bug report', please select OS:**  (check one with "x")
- [ ] **Android**
- [x] **iOS**

**cordova information:**  (run `$> cordova plugin list`) 

cordova-plugin-battery-status 1.2.4 "Battery"
cordova-plugin-console 1.0.7 "Console"
cordova-plugin-customurlscheme 4.3.0 "Custom URL scheme"
cordova-plugin-device 1.1.6 "Device"
cordova-plugin-device-orientation 1.0.7 "Device Orientation"
cordova-plugin-globalization 1.0.7 "Globalization"
cordova-plugin-googlemaps 2.2.4 "cordova-plugin-googlemaps"
cordova-plugin-hidden-statusbar-overlay 2.0.1 "HiddenStatusbarOverlay"
cordova-plugin-inappbrowser 2.0.2 "InAppBrowser"
cordova-plugin-inapppurchase 1.1.0 "In App Purchase"
cordova-plugin-insomnia 4.3.0 "Insomnia (prevent screen sleep)"
cordova-plugin-network-information 1.3.3 "Network Information"
cordova-plugin-splashscreen 4.0.3 "Splashscreen"
cordova-plugin-tts 0.2.3 "TTS"
cordova-plugin-whitelist 1.3.2 "Whitelist"
cordova-plugin-x-socialsharing 5.1.8 "SocialSharing"
cordova-universal-links-plugin 1.2.1 "Universal Links Plugin"
cordova.plugins.diagnostic 3.9.2 "Diagnostic"
es6-promise-plugin 4.2.2 "Promise"

**Current behavior:**
For iPhone X, the bottom of the app is completely unresponsive. I cannot pan the map, or click any buttons that are placed at the bottom. 

In the attached screenshot, you can see the placement of the Google logo is oddly high, and that area below it cannot be interacted with. 

The image on the right is from an iPhone 7, and it works as expected. Buttons appear at the bottom sometimes, and they can be clicked.

Edit: BTW, I'm not sure which version, but it's v2.0+ and this does not happen on the iPhone X. Something happened between 2.0 and now that caused this.

**Expected behavior:**
The bottom of the map should be draggable, and buttons placed there should be clickable.

**Screen capture or video record:**
![errorexample](https://user-images.githubusercontent.com/6970514/36457883-65ec78f8-1669-11e8-8bff-139149170a3c.jpg)

**Comment by wf9a5m75** [comment]
Please share your project files on github.

**Comment by caleb87** [comment]
I just created a new phonegap project, then added cordova-plugin-googlemaps, then used the demo code provided on the read me (with html/body at height/width of 100%). I noticed that if I set the map canvas to a fixed dimension, like 500px by 500px, then it displays correctly.  

I'm creating a demo repo I'll post in an hour or so if you really want it.

The index.html code is:



`<!DOCTYPE html>

<html>

<head>
    <meta charset="utf-8" />
    <meta name="format-detection" content="telephone=no" />
    <meta name="msapplication-tap-highlight" content="no" />
    <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width" />
    
    <meta http-equiv="Content-Security-Policy" content="default-src * 'unsafe-inline' gap:; style-src 'self' 'unsafe-inline'; media-src *" />
    
    

    <link rel="stylesheet" type="text/css" href="css/index.css" />
    <title>Hello World</title>
    <style>
        html {
            height: 100%; width: 100%;
        
        }
        body {
            
            padding: 0px; margin: 0px; height: 100%; width: 100%;
        }
        </style>
</head>

<body>

<div style="width: 100% !important; height: 100% !important;" id="map_canvas"></div>
    <script type="text/javascript" src="cordova.js"></script>

        <script type="text/javascript">
        var map;
        document.addEventListener("deviceready", function() {
                                  alert('ready');
                                  var div = document.getElementById("map_canvas");
                                  
                                  // Initialize the map view
                                  map = plugin.google.maps.Map.getMap(div);
                                  
                                  // Wait until the map is ready status.
                                  map.addEventListener(plugin.google.maps.event.MAP_READY, onMapReady);
                                  }, false);
                                  
                                  function onMapReady() {
                                      var button = document.getElementById("button");
                                      button.addEventListener("click", onButtonClick);
                                  }
    
    function onButtonClick() {
        
        // Move to the position with animation
        map.animateCamera({
                          target: {lat: 37.422359, lng: -122.084344},
                          zoom: 17,
                          tilt: 60,
                          bearing: 140,
                          duration: 5000
                          }, function() {
                          
                          // Add a maker
                          map.addMarker({
                                        position: {lat: 37.422359, lng: -122.084344},
                                        title: "Welecome to \n" +
                                        "Cordova GoogleMaps plugin for iOS and Android",
                                        snippet: "This plugin is awesome!",
                                        animation: plugin.google.maps.Animation.BOUNCE
                                        }, function(marker) {
                                        
                                        // Show the info window
                                        marker.showInfoWindow();
                                        
                                        // Catch the click event
                                        marker.on(plugin.google.maps.event.INFO_CLICK, function() {
                                                  
                                                  // To do something...
                                                  alert("Hello world!");
                                                  
                                                  });
                                        });
                          });
    }
    </script>
</body>

</html>
`

**Comment by caleb87** [comment]
Repo that demonstrates the bug: https://github.com/caleb87/googlemapsdemo

**Comment by wf9a5m75** [comment]
![screen shot 2018-02-20 at 10 30 55 pm](https://user-images.githubusercontent.com/167831/36466214-bf3c15d4-168d-11e8-89d9-692534d26858.png)

**Comment by wf9a5m75** [comment]
I can not see any marker cluster code in this file.
https://github.com/caleb87/googlemapsdemo/blob/f005664cee430fe2f1a32b12a7580540a4296b11/www/index.html

**Comment by wf9a5m75** [comment]
![2098](https://user-images.githubusercontent.com/167831/36467176-6ffd384a-1692-11e8-8a68-3dfd4b93f107.gif)

**Comment by wf9a5m75** [comment]
Google logo is managed by the native sdk, not this plugin.

**Comment by caleb87** [comment]
I apologize as I was wrong. I literally rebuilt the app 50+ times and the bottom below the Google logo was unresponsive. I removed these four plugins at same time, and then it worked. I went back to find the culprit by adding the exact version, and now I cannot get the bug to reproduce. 

cordova-plugin-statusbar 2.1.3 "StatusBar"
cordova-plugin-contacts 2.0.1 "Contacts"
cordova-plugin-device-motion 1.2.5 "Device Motion"
cordova-plugin-file-transfer 1.5.1 "File Transfer"

:(
